org.eclipse.january.test/src/org/eclipse/january/dataset/DatasetUtilsTest.java

```diff
@@ -0,0 +1,23 @@
+/*-
+ * Copyright (c) 2017 Kichwa Coders Ltd. and others
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.january.dataset;
+
+import org.junit.*;
```

Comment:
Run Organize Imports (Ctrl-Shift-o)  on this to remove the .* see http://javadude.com/articles/importondemandisevil.html for more info